Scale InfluxDB Clustered (#5526)
influxdata/docs-v2: content/influxdb/clustered/reference/glossary.md

## W

### WAL (Write-Ahead Log)

The temporary cache for recently written points.
To reduce the frequency that permanent storage files are accessed, InfluxDB
caches new points in the WAL until their total size or age triggers a flush to
more permanent storage. This allows for efficient batching of the writes into
the storage engine.

> Points in the WAL can be queried and persist through a system reboot.

Reviewer: ```suggestion
Points in the WAL are queryable and persist through a system reboot.
```
